recipes/nim-falcon/meta.yaml

```diff
@@ -9,22 +8,29 @@ package:
 extra:
   recipe-maintainers:
     - pb-cdunn
+  skip-lints:
+    - should_be_noarch_generic # We use "skip", so this is a flawed lint-check.
 
 source:
-  url: https://github.com/bio-nim/{{ name }}/releases/download/{{ version }}/{{ name }}-osx-64.tar.gz # [osx]
-  url: https://github.com/bio-nim/{{ name }}/releases/download/{{ version }}/{{ name }}-linux-64.tar.gz # [linux and x86_64]
-  sha256: 7d7e25ce05aee198273a6816dad52a893a7b07f778fba926b7428c9ec92b6662 # [osx]
-  sha256: 7831fb9a10d90b67c507d1b57b8d9dfe5ffb701bb8901252cc93c79ffc8d5aa4 # [linux and x86_64]
+  url: https://github.com/bio-nim/{{ name }}/releases/download/{{ version }}/{{ name }}-linux-64.tar.gz
+  sha256: 605ca8edbaafe85d24c2d1cb40885c0be7b82196550c17b4da7d1c529637c28f
 
 build:
+  skip: True # [osx]
   number: 0
-  name: prebuilt
+
+requirements:
+  host:
+    - zlib
+    - htslib
```

Comment:
don't need these at build-time?

Reply:
Really? I don't see how the test could pass without them.

Reply:
`host` is for build-time dependencies (and `test:` is a build-time phase)